html/forgot-password/index.php

```diff
@@ -0,0 +1,40 @@
+<?php
+session_start();
+require_once('../vendor/autoload.php');
+if ($_SERVER["REQUEST_METHOD"] == "POST" && isset($_POST["submit"])) {
+  $obj = new ForgotPassword($_POST["user_id"]);
+}
+?>
+
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>fill details</title>
+  <link rel="stylesheet" href="../css/style.css">
+  <link rel="stylesheet" href="../css/style_navbar.css">
+  <style>
+  </style>
+</head>
+
+<body>
+  <section class="details">
+    <div class="container">
+      <h1>welcome! please fill the details to move forward .</h1>
+      <form method="POST" action="index.php" enctype="multipart/form-data">
+        <div class="input-field mail">
+          <span>USER ID :</span> <input required type="text" name="user_id"
+            value="<?php echo isset($_POST['user_id']) ? $obj->user_id : '' ?>" placeholder="enter your user id">
```

Comment:
I already comment on this you can check it and please try to modify everywhere. Here you want to print the `user_id` when it will set then simple write like this `if(isset($_POST['user_id'])){echo $obj->user_id}` in this case `echo` statement will be execute only when `$_POST['user_id'])` is set. If you want to share something then please share your opinion.

Reply:
okay , got your point . try to change in all the files .